pysc2/lib/renderer_human.py

```diff
@@ -617,6 +630,27 @@ def draw_units(self, surf):
     """Draw the units and buildings."""
     for u, p in self._visible_units():
       if self._camera.intersects_circle(p, u.radius):
+        screen_loc = surf.world_to_surf.fwd_pt(p).floor()
+        aligned_screen_loc = point.Point(screen_loc.x - (self.scale * u.radius * 1.3),
```

Comment:
This looks sketchy... It shouldn't be needed if you use self.world_to_surf, and also seems to put it off center. I suspect this also locks it very much to your screen resolution. Please test different resolutions with ctrl++ and ctrl+-.